Rely on the compiler to be properly setup for the default ABI.

When installing-headers, there are two cases:
- NPTL: no issue, a cross-compiler is already expected
- LinuxThreads: no issue, EABI/OABI has no impact on installed headers.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
Cc: Khem Raj <raj.khem@gmail.com>
Cc: Bernhard Reutner-Fischer <rep.dot.nop@gmail.com>
Cc: Carmelo AMOROSO <carmelo.amoroso@st.com>
---

--- a/extra/Configs/Config.arm
+++ b/extra/Configs/Config.arm
@@ -12,17 +12,6 @@ config FORCE_OPTIONS_FOR_ARCH
 	default y
 	select ARCH_ANY_ENDIAN
 
-config CONFIG_ARM_EABI
-	bool "Build for EABI"
-	help
-	  If you say 'y' here, functions and constants required by the
-	  ARM EABI will be built into the library.  You should say 'y'
-	  if your compiler uses the ARM EABI, in which case you will also
-	  need a kernel supporting the EABI system call interface.
-
-	  If you say 'n' here, then the library will be built for the
-	  old Linux ABI.
-
 config COMPILE_IN_THUMB_MODE
 	bool "Build using Thumb mode"
 	select USE_BX
